lib/apollo-federation/schema.rb

```diff
@@ -47,7 +47,7 @@ def to_graphql
         super
       end
 
-      def federation_sdl(context: nil)
```

Comment:
Let's change this back to an optional keyword arg and update `ServiceField` to pass in `context` as a keyword arg.